# Request saving a list of columns for the map identified by map_uuid.
# If no annotations are found on database for map_uuid, it does nothing.

# Unique ID of the associated map.
string map_uuid

yocs_msgs/ColumnList columns
---
# Whether annotations were found on database for map_uuid
bool found

# What went wrong, if anything
string error_msg
